docs/src/arrays.md

```diff
@@ -0,0 +1,440 @@
+# Deriving Array Rules
+
+When the inputs and outputs of the functions are arrays (potentially with scalars), a modified version of Giles' method is a quick way of deriving [`frule`](@ref)s and [`rrule`](@ref)s.
```

Comment:
Usually the second `s` is added if it would be pronouned (which i think it would be for "Giles's method"

```suggestion
When the inputs and outputs of the functions are arrays (potentially with scalars), a modified version of Giles's method is a quick way of deriving [`frule`](@ref)s and [`rrule`](@ref)s.
```